docs/getting-started/setup.md

```diff
@@ -3,6 +3,9 @@ id: setup
 title: Technical Setup
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ## Requirements
 
 To run this tutorial you just need two things:
```

Comment:
I suggest to replace 
If you use Windows, please read our note on [Docker for Windows](troubleshooting/docker-on-windows.md),
with

If you use Windows, you might want to use [podman as an alternative to docker](troubleshooting/docker-on-windows.md).

Reply:
great suggestion. Thanks. Changed.